module-bsp/board/linux/torch/torch.cpp

```diff
@@ -1,65 +1,62 @@
-// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #include "bsp/torch/torch.hpp"
 #include <log/log.hpp>
 
-namespace bsp
+namespace bsp::torch
 {
-    namespace torch
+    State state_simulated               = State::Off;
+    ColourTemperature currentColourTemp = ColourTemperature::Warmest;
+
+    std::int32_t init()
     {
-        State state_simulated               = State::off;
-        ColourTemperature currentColourTemp = ColourTemperature::warmest;
+        state_simulated = State::Off;
+        return 1;
+    }
 
-        std::int32_t init()
-        {
-            state_simulated = State::off;
-            return 1;
-        }
+    void deinit()
+    {}
 
-        void deinit()
-        {}
+    bool isPresent()
+    {
+        return true;
+    }
 
-        bool isPresent(void)
-        {
-            return true;
+    bool turn(State state, ColourTemperature colourTemp)
+    {
+        state_simulated = state;
+        if (colourTemp != ColourTemperature::NoChange) {
+            currentColourTemp = colourTemp;
         }
 
-        bool turn(State state, ColourTemperature colourTemp)
-        {
-            state_simulated = state;
-            if (colourTemp != ColourTemperature::noChange) {
-                currentColourTemp = colourTemp;
-            }
-
-            if (state == State::on) {
-                LOG_INFO("Torch is ON \U0001f526 (%s)",
-                         currentColourTemp == ColourTemperature::warmest ? "warm \U0001f987" : "cold \U0001f535");
-            }
-            else {
-                LOG_INFO("Torch is OFF");
-            }
-            return true;
+        if (state == State::On) {
+            LOG_INFO("Torch is ON \U0001f526 (%s)",
+                     currentColourTemp == ColourTemperature::Warmest ? "warm \U0001f987" : "cold \U0001f535");
         }
-
-        State getState()
-        {
-            return state_simulated;
+        else {
+            LOG_INFO("Torch is OFF");
         }
+        return true;
+    }
 
-        ColourTemperature getColorTemp()
-        {
-            return currentColourTemp;
-        }
+    State getState()
+    {
+        return state_simulated;
```

Comment:
What about camel case convention instead of snake here ;)

Reply:
Done :wink: